src/components/Form/FormSection.jsx

```diff
@@ -48,12 +39,6 @@ const FormSection = ({
     </Typography>
     {children}
     {showSpinner && <CircularProgress />}
```

Comment:
Still a bit confused over the spinner here. I'm not sure where it's being used when going over the notifications.

Reply:
The spinner was taken out when I was trying to separate the implementation of the notifications from the PR where the system was created.

Either way it appeared in the modal while the async function was running. I feel the spinner is a better solution that a message because of how quickly it happens. Flashing a message too quickly for the user to read it is not needed if a loading spinner can tell them that something is happening. 

I do think the spinner could be deleted in this issue and I could set up a separate one which might be a better Idea if Form Section is going away.